### DISCUSSION TITLE
Anyone else seeing MD5 hash on Windows msi and exe for 8.5.7 not agreeing with what is posted on cyberduck webpage?

### DISCUSSION THREAD

**Discussion by weberhans** [iterate-ch/cyberduck#14459] [Q&A]
https://update.cyberduck.io/windows/Cyberduck-Installer-8.5.7.39535.exe
and
https://update.cyberduck.io/windows/Cyberduck-Installer-8.5.7.39535.msi

Website says for .exe
Version 8.5.7, 22 Mar 2023
MD5 8b5a7d5031da9e9fdf616d99d0b0efd4

I get ...

C:\> certutil -hashfile C:\Users\hxw\Downloads\Cyberduck-Installer-8.5.7
.39535.exe MD5
MD5 hash of file C:\Users\hxw\Downloads\Cyberduck-Installer-8.5.7.39535.exe:
0d de 34 b9 a5 8c 63 17 89 20 66 df c5 af 8e 11
CertUtil: -hashfile command completed successfully.

**Comment by dkocher** [comment]
There was an issue with the release build being run twice. The actual checksum has been fixed in iterate-ch/cyberduck.io#1 and the correction has just been pushed now.
